core/src/apps/cardano/sign_tx.py

```diff
@@ -4,32 +4,42 @@
 from trezor.crypto import base58, hashlib
 from trezor.crypto.curve import ed25519
 from trezor.messages.CardanoSignedTx import CardanoSignedTx
-from trezor.messages.CardanoTxAck import CardanoTxAck
-from trezor.messages.CardanoTxRequest import CardanoTxRequest
 
-from apps.cardano import CURVE, seed
+from apps.cardano import CURVE, MAINNET_PROTOCOL_MAGIC, TESTNET_PROTOCOL_MAGIC, seed
 from apps.cardano.address import (
     derive_address_and_node,
     is_safe_output_address,
     validate_full_path,
 )
-from apps.cardano.layout import confirm_sending, confirm_transaction, progress
+from apps.cardano.layout import confirm_sending, confirm_transaction
 from apps.common import cbor
 from apps.common.paths import validate_path
 from apps.common.seed import remove_ed25519_prefix
 
+if False:
```

Comment:
forgotten?

Reply:
No, this is how types, which are only imported for the sake of type hinting, are imported throughout the project.